### ISSUE TITLE
Hasher implementation is required to decode SD-JWT

### ISSUE THREAD

**Issue by DeanDeBlock** [Sphereon-Opensource/mobile-wallet#338]
Hi all,

Sphereon wallet is having issues interpreting my sd-jwt formatted vc

![IMG_A69AAA17A69B-1](https://github.com/user-attachments/assets/16a4e963-98ff-41fe-bec6-8f6e8f84a276)

Any ideas what's missing here? This is the vc the wallet receives:

`eyJ0eXAiOiJKV1QiLCJhbGciOiJFUzI1NiIsImtpZCI6ImRpZDprZXk6ejJkbXpEODFjZ1B4OFZraTdKYnV1TW1GWXJXUGdZb3l0eWtVWjNleXFodDFqOUticVdRMUFDU0F0dk5FMWJSY0ZLNzdtV29peXRaVnJ4RjUyWWVVR01vRWtvaHhDdFNRNDNQaVpLa3BmUG5IVm5yeGJ2Y0hYV2lZMTFwWHhGWHQ4TkRKTDdiNG9reFFEMXNzcm5OeWlnN1ZHN3Voc1hXbUJXN0dxZDJEdWhvRHpCcjdmYSN6MmRtekQ4MWNnUHg4VmtpN0pidXVNbUZZcldQZ1lveXR5a1VaM2V5cWh0MWo5S2JxV1ExQUNTQXR2TkUxYlJjRks3N21Xb2l5dFpWcnhGNTJZZVVHTW9Fa29oeEN0U1E0M1BpWktrcGZQbkhWbnJ4YnZjSFhXaVkxMXBYeEZYdDhOREpMN2I0b2t4UUQxc3Nybk55aWc3Vkc3dWhzWFdtQlc3R3FkMkR1aG9EekJyN2ZhIn0.eyJ2Y3QiOiJTRC1KV1QtQ1RXYWxsZXRTYW1lQXV0aG9yaXNlZEluVGltZSIsImNyZWRlbnRpYWxTdWJqZWN0IjoiIiwiaWQiOiJ1cm46c2VydmljZV9wcm92aWRlciIsIl9zZCI6WyJvUFU0VExNR05CejBnYURPcVpJUFotYnV0WE9VRXZlYmxzdHQ2bDF3TTQ4IiwieEF3ZGVZR1A3ZmpvTE51LVU0bHg2S0lOZWdjak8xMDR0MU0yU09BYmxLMCIsInBvYm1VR1ljcXoybl9Wd2ZjVXcwY2swN1MtZ0tMT2xuVC1UQWxJdHpQLTQiLCJKcnhpOUQyU1hQamdTeDlNVlIwWVNRa0lFck1ycHNGdFFNdzBFQ2haNE5JIiwib3cwZV9rVHBadE9IaVlkSWpkRDVDUmtTU1M0M0dBSG1hOEhCX1k2U20yRSJdLCJpc3N1YW5jZURhdGUiOiIyMDI0LTEyLTAyVDExOjQ2OjA2WiIsImlzc3VlZCI6IjIwMjQtMTItMDJUMTE6NDY6MDZaIiwidmFsaWRGcm9tIjoiMjAyNC0xMi0wMlQxMTo0NjowNloiLCJ2YWxpZFVudGlsIjoiMjAyNi0xMi0wMlQxMTo0NjowOVoiLCJfc2RfYWxnIjoic2hhLTI1NiIsImNuZiI6eyJqd2siOnsia3R5IjoiRUMiLCJjcnYiOiJQLTI1NiIsIngiOiJUQ0FFUjE5WnZ1M09IRjRqNFc0dmZTVm9ISVAxSUxpbERsczd2Q2VHZW1jIiwieSI6Ilp4amlXV2JaTVFHSFZXS1ZRNGhiU0lpcnNWZnVlY0NFNnQ0alQ5RjJIWlEifX0sImlzcyI6ImRpZDprZXk6ejJkbXpEODFjZ1B4OFZraTdKYnV1TW1GWXJXUGdZb3l0eWtVWjNleXFodDFqOUticVdRMUFDU0F0dk5FMWJSY0ZLNzdtV29peXRaVnJ4RjUyWWVVR01vRWtvaHhDdFNRNDNQaVpLa3BmUG5IVm5yeGJ2Y0hYV2lZMTFwWHhGWHQ4TkRKTDdiNG9reFFEMXNzcm5OeWlnN1ZHN3Voc1hXbUJXN0dxZDJEdWhvRHpCcjdmYSIsImp0aSI6InVybjpzZXJ2aWNlX3Byb3ZpZGVyIiwic3ViIjoic2VydmljZV9wcm92aWRlciIsIm5iZiI6MTczMzEzOTk2NiwiZXhwIjoxNzMzMjI2MzY5LCJpYXQiOjE3MzMxMzk5NjYsImFsZyI6IkVTMjU2IiwidHlwIjoidmMrc2Qtand0In0.26AWO6sqBOOeA4I0aemMuAImIXP0ojYyHEuo4VXOGESg6nQ0FIpOKmmqxvlGmW2cBnupGc0T0oqp2A_t8cJRIQ~WyJHcll0RVJHdlJtSkJZZ1d0V2Z4MEFRIiwiZmlyc3ROYW1lIiwiRGVhbiJd~WyI1ZzN0bE5nTktsUkU4S0J3REFoVDJRIiwibWlkZGxlTmFtZSIsbnVsbF0=~WyJqS0p5dWtyTmpteTJyMGpwbHpqZ2xBIiwiZ2VuZGVyIixudWxsXQ==~WyJWbXJSNWJMcG9TQTMtUEl2N1B5NTFBIiwibGFzdE5hbWUiLCJEZSBCbG9jayJd~WyI5T1J0STJuS2h3Zm9tSjNmOGFWZjVBIiwiZW1haWxBZGRyZXNzIixudWxsXQ==~`

**Comment by sanderPostma** [comment]
Hi Dean,
This has been fixed in a newer release. Is the gmail in your contact also your play store account? I can add you to the pre-release list.

**Comment by DeanDeBlock** [comment]
Yes, that would be great. My gmail is indeed being used for iOS app store
